openmetadata-ui/src/main/resources/ui/src/styles/components/table.less

```diff
@@ -62,6 +62,11 @@
             border-bottom: none;
           }
         }
+        // Removed padding for expanded rows to prevent unnecessary scrolling behavior for the expanded row content
+        tbody > .ant-table-expanded-row > td {
```

Comment:
This looks risky, can we keep it to a specific table for now? will add it globally in 1.1

Reply:
Sure, I'll make the chhanges.